src/style/custom.css

```diff
@@ -20,8 +20,15 @@
        width: 293px;
        height: 409px;
        right: 0;
-       top: 30px;
+       top: 20px;
     } 
+    .sm\:bg-oval{
+       background-image: url('../static/svg/ovals.svg');
+       width: 251px;
+       height: 301px;
+       top: 20px;
+    }
+
```

Comment:
the other classname selectors don't have empty lines between them. if that's the syntax convention, then remove this empty line